pkg/event/ptp/syncstate.go

```diff
@@ -12,36 +12,61 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package event
+package ptp
 
 // SyncState ...
-type SyncState string
+type SyncState int
 
+// start with 1 so we can check against unitialized value 0
 const (
 	//LOCKED ...
-	LOCKED SyncState = "LOCKED"
+	LOCKED SyncState = iota + 1
```

Comment:
don't have to use iota, instead use string.

Reply:
done